### PR TITLE
fix(docs): the environment variables file should be called .env.local…

### PR DESCRIPTION
**Pull request by ItsWendell**
… in Next.js

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ To try out this repo yourself, follow the steps below:
 
 1. Clone the repo to your computer (`git clone git@github.com:payloadcms/payload-3.0-alpha-demo.git`)
 1. `cd` into the new folder by running `cd ./payload-3.0-alpha-demo`
-1. Copy the `.env.example` by running `cp .env.example .env` in the repo, then fill out the values including the connection string to your DB
+1. Copy the `.env.example` by running `cp .env.example .env.local` in the repo, then fill out the values including the connection string to your DB
```
Comment (franciscolourenco) [inline README.md:51]:
```suggestion
1. Copy the `.env.local.example` by running `cp .env.local.example .env.local` in the repo, then fill out the values including the connection string to your DB
```

**Comment by jacobsfletch** [comment]
Related docs: https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables

**Comment by jacobsfletch** [comment]
Hey @ItsWendell and @franciscolourenco thanks for bringing this up 👍 we also need to _rename_ the .env.example file itself to `.env.local.example` _and_ ensure that this new env path is respected on the Payload-side, i.e. connection string, etc.

**Comment by AlessioGr** [comment]
Closing, as this has been resolved. Support for .env.local has been added, and the docs have been adjusted

**Pull request closed without merging**